pennylane/devices/tests/test_gates.py

```diff
@@ -421,6 +421,8 @@ def test_two_qubit_no_parameters(self, device, init_state, op, mat, tol, skip_if
         n_wires = 2
         dev = device(n_wires)
         skip_if(dev, {"returns_probs": False})
+        if not dev.supports_operation(op):
```

Comment:
Frankly, I'm still conflicted with having these statements in when we also have a dedicated mark to doing this. Down the line, it would be great to investigate why the mark was not working because now this will likely lead to confusion in the future as to why we added these statements and have the mark too.